tests/opt/opt_expr_alu.ys

```diff
@@ -60,7 +60,8 @@ alumacc
 equiv_opt -assert opt_expr -fine
 design -load postopt
 select -assert-count 1 t:$pos
-select -assert-count none t:$pos t:* %D
+select -assert-count 1 t:$not
+select -assert-count none t:$pos t:$not %% t:* %D
```

Comment:
Wouldn't it be slightly terser to use `-assert-none` here?

Reply:
Actually, `-assert-count none` relies on `none` being interpreted as `0`:
https://github.com/YosysHQ/yosys/blob/3c4758c60e33386e19049d81fd6f72c8e7f316e0/passes/cmds/select.cc#L1265-L1268
Sorry, another issue from copy-pasting from my tests.
So either `-assert-count 0` for spacing, or `-assert-none` as @ZirconiumX suggests.